src/serializers/decoders/current/v19/decode_graph_schema.c

```
@@ -0,0 +1,324 @@
+			char *stopword = SerializerIO_ReadBuffer(rdb, NULL);
+			arr_append(stopwords, stopword);
+		}
+	}
+
+	uint fields_count = SerializerIO_ReadUnsigned(rdb);
+	for(uint i = 0; i < fields_count; i++) {
+		IndexFieldType type;
+		double         weight;
+		bool           nostem;
+		char*          phonetic;
+		char*          field_name;
```

Cppcheck (2.20.0)

[error] Common realloc mistake

(memleakOnRealloc)